### DISCUSSION TITLE
Where to ask questions

### DISCUSSION THREAD

**Discussion by nathanielvirgo** [probcomp/Gen.jl#505] [Q&A] [answered]
Sorry, this isn't really an issue. I'm starting to learn gen but I can't figure out where I should ask basic beginner questions. Is there a forum or mailing list somewhere?

(To make this an actual issue: if there is such a resource, it would be helpful to update the page at gen.dev to point to it.)

**Comment by femtomc** [comment]
@nathanielvirgo in the past, people have asked questions in issues! And mostly, developers for Gen respond there.

However, perhaps it is worth exploring another venue for this now.

@alex-lew @ztangent -- I wonder if "GitHub discussions" might be an appropriate way to organize non-issues, but questions.

**Comment by femtomc** [comment]
Re -- one philosophical thing: I think if a question is sufficiently important (and broad) that it is likely to be asked over and over again, the documentation should be the right place for the answer. So one process for including new information which arises from issues is to have a discussion on issue, then setup a PR for the documentation which includes that new information.

**Comment by alex-lew** [comment]
Hi @nathanielvirgo, thanks for trying out Gen! Github issues are one place you can ask. The Julia Slack also has a #gen channel that you can use to ask Gen-specific questions, and a #probprog channel for more general questions about probabilistic programming.

**Comment by ztangent** [comment, marked as answer]
I just created a Github discussions page for this repository, so hopefully questions can go there in the future!
